build-scripts/hack/sync-registry.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+docker run -v "${GITHUB_WORKSPACE}/.github/data/sync-registry-config.yaml":/config.yaml quay.io/skopeo/stable:v1.15 sync \
```

Comment:
This uses `${GITHUB_WORKSPACE}`. If run on a local machine, it would try to find `/.github/data/sync-registry-config.yaml` which is probably not what we want.